src/pages/AddItem.js

```diff
@@ -29,17 +51,15 @@ const AddItem = ({ firestore }) => {
     setNextExpectedPurchase(parseInt(event.target.value, 10));
   };
 
-  // Handle the click of the Add Item button on the form
   const handleSubmit = event => {
     event.preventDefault();
-    addItem(name, token);
+    addItem();
     setName('');
```

Comment:
I think it would be a slightly better user experience for the value the user input to remain in the case where you show the error. It could be they made a typo or want to enter the same thing but a different size.

I'd like to see this one addressed if you have time before tomorrow. Thanks!